website/docs/foundations/focus-ring/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Focus ring
+caption: HDS - “All interactive elements must be able to receive focus and focus must be visible.”
 ---
```

Comment:
This was copied from our Figma. This is just an example of something more practical-sounding language but not intended as a practical final example...